src/objects/Education_History__c.object

```diff
@@ -0,0 +1,632 @@
+                    <fullName>100</fullName>
+                    <default>false</default>
+                    <label>100</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>GPA_Scale_Type__c</fullName>
+        <description>Tracks how the GPA was determined. For example, using a weighted or unweighted scale.</description>
+        <externalId>false</externalId>
+        <label>GPA Scale Type</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Unweighted</fullName>
+                    <default>false</default>
+                    <label>Unweighted</label>
+                </value>
+                <value>
+                    <fullName>Weighted</fullName>
+                    <default>false</default>
+                    <label>Weighted</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>GPA__c</fullName>
+        <description>The Contact&apos;s final grade point average.</description>
+        <externalId>false</externalId>
+        <label>GPA</label>
+        <precision>7</precision>
+        <required>false</required>
+        <scale>4</scale>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Graduation_Date__c</fullName>
+        <description>If applicable, the date the Contact graduated from this Educational Institution.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>If applicable, the date the Contact graduated from this Educational Institution.</inlineHelpText>
+        <label>Graduation Date</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <fields>
+        <fullName>Start_Date__c</fullName>
+        <description>The date the Contact started at this Educational Institution.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The date the Contact started at this Educational Institution.</inlineHelpText>
+        <label>Start Date</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <fields>
+        <fullName>Time_At_Institution__c</fullName>
+        <description>The total amount of time a Contact spent at the Educational Institution, determined by the Start Date and End Date.</description>
+        <externalId>false</externalId>
+        <formula>IF(
+    NOT(ISBLANK(End_Date__c)) &amp;&amp; NOT(ISBLANK(Start_Date__c)) &amp;&amp; (End_Date__c > Start_Date__c),
```

Comment:
@m-kinkade While this solution is effective, it is not precise. It'll start to slip over large amounts of time. We may have to find some manner of compromise that excludes days as a significant figure.